### PR TITLE
Propagate artifact visibility

### PR DESCRIPTION
**Pull request by josenavas**
This emulates the prior functionality in which the visibility of a processed data was propagated to the previous objects.

### CODE REVIEW

File: qiita_pet/handlers/study_handlers/description_handlers.py [hunk 1]
```diff
@@ -84,6 +84,28 @@ def _to_int(value):
...
+    """Propagates the visibility of artifact to all its ancestors
+
+    Parameters
+    ----------
+    artifact : qiita_db.artifact.Artifact
+        The artifact to propagate the visibility for
+
+    Notes
+    -----
+    This is emulating the previous functionlity, in which the status of the
```
Comment (antgonza) [inline qiita_pet/handlers/study_handlers/description_handlers.py]: functionality

File: qiita_pet/test/test_study_handlers.py [hunk 2]
```diff
@@ -156,6 +159,20 @@ def test_build_study_info_new_study(self):
             'proc_data_info': []})
         self.assertEqual(obs, self.exp)
 
+    def test_propagate_visibility(self):
```
Comment (antgonza) [inline qiita_pet/test/test_study_handlers.py]: Is there a test for those that are not BIOM?

**Comment by antgonza** [comment]
2 comments

**Comment by antgonza** [comment]
2 comments

File: qiita_pet/handlers/study_handlers/description_handlers.py (hunk 1 above)
Comment (josenavas) [inline qiita_pet/handlers/study_handlers/description_handlers.py]: Done

File: qiita_pet/test/test_study_handlers.py (hunk 2 above)
Comment (josenavas) [inline qiita_pet/test/test_study_handlers.py]: Done

File: qiita_pet/handlers/study_handlers/description_handlers.py [hunk 3]
```diff
@@ -84,6 +84,28 @@ def _to_int(value):
     return res
 
 
+def _propagate_visibility(artifact):
+    """Propagates the visibility of artifact to all its ancestors
```
Comment (wasade) [inline qiita_pet/handlers/study_handlers/description_handlers.py]: "of artifact" -> "of an artifact"

**Comment by wasade** [comment]
One small comment, :+1: otherwise

**Comment by ElDeveloper** [comment]
:+1: granted that @wasade comment is addressed!

**Comment by mortonjt** [comment]
:+1: once the additional BIOM tests are added

File: qiita_pet/handlers/study_handlers/description_handlers.py (hunk 3 above)
Comment (josenavas) [inline qiita_pet/handlers/study_handlers/description_handlers.py]: Done

**Pull request merged**